Framework/Utils/JsonReader.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Framework.Utils
+{
+    public static class JsonReader
+    {
+        public static T SetConfigModel<T>() where T : class
+        { 
+            JObject jModel;
+            var assembly = Assembly.GetCallingAssembly();
+            var resourcePath = $"{assembly.GetName().Name}.Resources.{typeof(T).Name}.json";
```

Comment:
it is better to pass the file path as a parameter, this way our method will become more universal